rpc/common/moc_common_common.proto

```diff
@@ -324,6 +325,9 @@ enum ProviderAccessOperation {
 
 	NetworkSecurityGroup_Create 	= 3600;
 	NetworkSecurityGroup_Update 	= 3601;
+
+	AvailabilityZone_Create		    = 3700;
+	AvailabilityZone_Delete		    = 3701;
```

Comment:
Since Zone can be updated. Can you please check if we need 'AvailabilityZone_Update' too ?

Reply:
added avzone update enum